### PR TITLE
Vagrantfile: install pip by default

### PR DESCRIPTION
**Pull request by aalemayhu**
Required for installing the sphinx dependencies.

Signed-off-by: Alexander Alemayhu <alexander@alemayhu.com>

### CODE REVIEW

**Comment by aalemayhu** [comment]
Hmm, I am still seeing 
```bash
[Runtime Tests] ==> cilium-master: ./Documentation/_build/linkcheck/output.txt
[Runtime Tests] ==> cilium-master: : No such file or directory
```
IIRC, ../Documentation did not work and ./Documentation worked. 

@aanm Which one is correct for the CI https://github.com/cilium/cilium/blob/master/tests/00-documentation-link-check.sh#L3 ?

Thanks.

**Comment by aanm** [comment]
>IIRC, ../Documentation did not work and ./Documentation worked.
>
>@aanm Which one is correct for the CI https://github.com/cilium/cilium/blob/master/tests/00-documentation-link-check.sh#L3 ?

Where it didn't work and where it did work?

**Comment by tgraf** [comment]
> IIRC, ../Documentation did not work and ./Documentation worked.

`../Documentation` would seem correct as we are runnings tests with a `PWD=./tests`

**Comment by aalemayhu** [comment]
Suspect I mixed up local setup and vm earlier. Will revert 992d096614ad (tests: make link check ready for vagrant, 2017-05-30).

Thanks.

**Review by aanm — APPROVED** [review]

File: Vagrantfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ end
 
 $bootstrap = <<SCRIPT
 sudo apt-get -y update || true
-sudo apt-get -y install socat curl jq realpath pv tmux python-sphinx
```
Comment (aanm) [inline Vagrantfile:16]: @tgraf is there any reason why isn't this in our atlas VM?

**Pull request merged**